Wikipedia/Code/Panels.swift

```diff
@@ -193,6 +193,11 @@ extension UIViewController {
         }
         let panel = SyncEnabledPanelViewController(showCloseButton: true, primaryButtonTapHandler: primaryButtonTapHandler, secondaryButtonTapHandler: nil, dismissHandler: nil, theme: theme)
         let presenter = self.presentedViewController ?? self
+        if presenter is WMFThemeableNavigationController {
+            guard presenter.presentedViewController == nil else {
+                return
```

Comment:
Do you think an assert makes sense here or is it something we expect to happen?

Reply:
We expect it to happen. "Sync enabled" panel will be presented when we get a response from the server that sync is enabled for the account. We can be presenting "Your reading lists will be synced in the background" alert at that time.

Reply:
If you expect it to happen it's fine.